modules/CMakeLists.txt

```diff
@@ -25,6 +25,7 @@ ev_add_modules(
     PersistentStore
     PN532TokenProvider
     Setup
+    SerialCommHub
```

Comment:
Concerning the naming, is this really a universal serial communication hub? Looking at the at the manifest, it looks more like a ModbusMultiplexer

Reply:
Currently it supports modbus and raw access for similar protocols that are not really modbus. full raw could be added as well